### ISSUE TITLE
Images ref disappears when coping sheet

### ISSUE THREAD

**Issue by Golans** [ClosedXML/ClosedXML#415]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

e.g. 0.87.1

**What is the current behavior?**
Scenario: You have a worksheet with images and want to duplicate(copy) it.
IXLWorksheet CopyTo(String newSheetName)
After saving the workbook and opening the Excel file result in ref errors to source images in source worksheet.

Checked it deeper and saw that source images where written with 0 bytes.

Fix below...

**If the current behavior is a bug, please provide the steps to reproduce and 
if possible a minimal demo of the problem with a sample spreadsheet.**

**What is the expected behavior or new feature?**

**Did this work in previous versions of our tool?  Which versions?**

- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Code to Fix problem:**

File: **"XLWorkbook_Save.cs"** Line: **2559**

**current code:**
```c#
using (var stream = new MemoryStream())
{
                pic.ImageStream.CopyTo(stream);
                stream.Seek(0, SeekOrigin.Begin);
                imagePart.FeedData(stream);
}
```

**Fix:**
need to reset stream position for saving same image once again:
```c#
using (var stream = new MemoryStream())
{
                pic.ImageStream.Position = 0;
                pic.ImageStream.CopyTo(stream);
                stream.Seek(0, SeekOrigin.Begin);
                imagePart.FeedData(stream);
}
```

**Comment by igitur** [comment]
Would you mind submitting a pull request?

**Comment by Golans** [comment]
this is my first time submitting to git.
hopefully did right :)





Golan
+972-52-874-2595
TAGO-Solutions Ltd. <contact@tago-solutions.com>

[image: tago-solutions ltd.] <http://www.tago-solutions.com>

**Comment by igitur** [comment]
Closed via #417
